### PR TITLE
TD1712 Add collection bid creation example app

### PR DESCRIPTION
**Pull request by frankisawesome**
### Hi👋, please prefix this PR's title with:

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [ ] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary

# Detail and impact of the change

## Security

# Anything else worth calling out?

### CODE REVIEW

**Review by Sam-Jeston — APPROVED** [review]
This changeset, while large, is isolated to new sample apps in `examples/` and a patch to the func-tests to use the provided allowlist. No SDK code change is included here

File: examples/orderbook/fulfill-collection-bid-with-nextjs/README.md
Comment (zaidarain1) [inline examples/orderbook/fulfill-collection-bid-with-nextjs/README.md:1]: Could you give this README a small update to provide a brief overview of the example, similar to other examples? Something small like a few sentences should be enough 😄

File: examples/orderbook/fulfill-collection-bid-with-nextjs/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "fulfill-collection-bid-with-nextjs",
```
Comment (zaidarain1) [inline examples/orderbook/fulfill-collection-bid-with-nextjs/package.json:2]: Could you prepend the `@examples/` to the name

File: examples/orderbook/fulfill-collection-bid-with-nextjs/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "react-dom": "^18"
+  },
+  "devDependencies": {
+    "@types/node": "^20",
+    "@types/react": "^18",
+    "@types/react-dom": "^18",
+    "eslint": "^8",
+    "eslint-config-next": "14.2.14",
+    "postcss": "^8",
+    "tailwindcss": "^3.4.1",
```
Comment (zaidarain1) [inline examples/orderbook/fulfill-collection-bid-with-nextjs/package.json:26]: Tailwind doesn't seem to be used at all in this example. Is it safe to be removed, along with the tailwind config file? Is postcss still needed for biom3/react or any other purpose if tailwind isn't present? If not, that could be cleaned up too, along with its config file

**Review by zaidarain1 — CHANGES_REQUESTED** [review]

**Review by zaidarain1 — APPROVED** [review]

**Review by Sam-Jeston — DISMISSED** [review]

**Review by CodeSchwert — APPROVED** [review]

**Review by Sam-Jeston — APPROVED** [review]

**Pull request merged**